elyra/util/url.py

```diff
@@ -63,3 +65,16 @@ def send(self, req, **kwargs):
 
     def close(self):
         pass
+
+
+def get_verify_parm(default: bool = True) -> Union[bool, str]:
+    """
+    Returns a value for the 'verify' parameter of the requests.request
+    method (https://requests.readthedocs.io/en/latest/api/). The value
+    is determined as follows: if environment variable CA_CERT_BUNDLE_PATH
+    is defined, use its value, otherwise return the default value.
+    """
+    if os.environ.get("CA_CERT_BUNDLE_PATH"):
```

Comment:
seems like os.environ.get returns None if the env is not present, though I have not tried.

https://stackoverflow.com/questions/16924471/difference-between-os-getenv-and-os-environ-get

So in any place you check if the env is present, you should do

https://www.codegrepper.com/code-examples/python/check+if+env+variable+exists+python
https://stackoverflow.com/questions/16924471/difference-between-os-getenv-and-os-environ-get
e.g. 

if os.environ.get("TRUSTED_CA_BUNDLE_PATH") is not None:

Reply:
I've changed the condition to ignore the environment variable if one of these conditions is met:
 - environment variable is set to the empty string: testing showed that setting `verify` to the empty string disables certificate validation, which isn't desirable
 - environment variable is set to white spaces only: the string is treated as a non-existing path, causing verification to fail